.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,29 @@
+name: "Prettier Check"
+
+on:
+  pull_request:
+
+jobs:
+  check-prettier:
+    name: Prettier Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download repo
+        uses: actions/checkout@v4
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Check code with Prettier
+        run: pnpm exec prettier --check .
```

Comment:
It would be better if we could tell people that their code was formatted by Prettier, that way people don't get confused and we can confirm workflow actually is doing work on the PR.